src/main/java/com/factly/dega/service/impl/OrganizationServiceImpl.java

```diff
@@ -152,4 +160,40 @@ public void delete(String id) {
             .map(organizationMapper::toDto);
     }
 
+    /**
+     * Get the organizations by keycloakId and pageable.
+     *
+     * @param keycloakId the keycloakId of the User
+     * @param pageable the pagination information
+     *
+     * @return List<OrganizationDTO> List of Organizations by keycloakId and pageable
+     */
+    public List<OrganizationDTO> getOrganizations(String keycloakId, Pageable pageable){
```

Comment:
rename keycloakId to keycloakUserId as it can be confused for organization's keycloakId